fastlane/spec/action_metadata_spec.rb

```diff
@@ -3,6 +3,10 @@
 describe Fastlane::Action do
   Fastlane::ActionsList.all_actions do |action, name|
     describe name do
+      before do
+        allow(FastlaneCore::FastlaneFolder).to receive(:path).and_return("./fastlane")
```

Comment:
@mgrebenets 

I created the PR!
Could you take a look at this? https://github.com/fastlane/fastlane/pull/8484